llvm/test/CodeGen/PowerPC/gcov_ctr_ref_init.ll

```diff
@@ -12,19 +12,11 @@ target triple = "powerpc-ibm-aix"
 ; CHECK-NEXT: L..__llvm_covinit_functions:
 ; CHECK-NEXT:     .vbyte  4, __llvm_gcov_writeout[DS]
 ; CHECK-NEXT:     .vbyte  4, __llvm_gcov_reset[DS]
+; The first .csect directive below is specifying the content of the csect.
+; The second .csect directive below is used to insert the .ref pseudo
+; instruction.
+; CHECK:    .csect __llvm_gcov_ctr_section[RW],3
```

Comment:
@syzaara:
The content of the csect is significant (and should be checked, not just skipped).
The change that added this test was responsible for creating the section so that references to the counters would, in turn, cause the initialization symbols to be kept.

Reply:
Posted https://github.com/llvm/llvm-project/pull/117577 to address this